application/src/main/java/com/sanction/thunder/email/EmailModule.java

```diff
@@ -0,0 +1,33 @@
+package com.sanction.thunder.email;
+
+import com.amazonaws.services.simpleemail.AmazonSimpleEmailService;
+import com.amazonaws.services.simpleemail.AmazonSimpleEmailServiceClientBuilder;
+import dagger.Module;
+import dagger.Provides;
+
+import javax.inject.Singleton;
+
+@Module
+public class EmailModule {
+
+  /**
+   * Supplies singleton AmazonSimpleEmailService instance for dependency injection.
+   *
+   * @return An AmazonSimpleEmailService instance.
+   */
+  @Singleton
+  @Provides
+  public AmazonSimpleEmailService provideAmazonSimpleEmailService() {
+    return AmazonSimpleEmailServiceClientBuilder
+        .standard()
+        .withRegion("us-east-1")
+        .build();
```

Comment:
I think I know how you can get rid of the Javadoc here, just move `.standard()` to line 21 and start the newline with `.withRegion()`. Then it'll be three lines and won't require a javadoc.

Reply:
Cool!